doc/introduction/unsupported_gradients.rst

```diff
@@ -117,34 +117,22 @@ In particular, the following code works as expected:
 >>> print_grad()
 [-0.09983342]
 
-But the following code raises an error:
+PennyLane can now use adjoint differentation for any state based measurement on ``default.qubit``. ``lightning.qubit``

```

Comment:
```suggestion
Adjoint differentation supports any state based measurement on ``default.qubit``. ``lightning.qubit``
```

Reply:
The reason for this change is that in a sense documentation exists 'outside' of time, so 'now' is not necessary here.